### ISSUE TITLE
Vue js ^3.0 + tailwind css >=2.0 examples; Garner 10+ votes or Donos to kickstart Laravel Inertia/Nuxt/Vue/React Native demo development

### ISSUE THREAD

**Issue by syntaxlexx** [lexxyungcarter/laravel-5-messenger#14]

**Comment by cjlaborde** [comment]
First vote, would like as well if you could add nuxt.js support as well.
Is something the other packages are not doing.
I use vue.js and tailwind together so looking forward to it.

**Comment by syntaxlexx** [comment]
Oh yeah. Nuxt.js is definitely on my mind as well. Well, between the votes and sponsors, whichever reaches first; and I'll kick-start development for it.
Am also thinking of a Livewire/Inertia.js one :)

**Comment by cjlaborde** [comment]
Focus on Nuxt.js  Since a lot of bigger projects are build on it. Especially commercial.
I don't care about  Livewire/Inertia.js, to be honest.
Can your project be used as API only?

**Comment by syntaxlexx** [comment]
Yeah definitely. I got one mobile app consuming it as API. Nuxt is very nice.

**Comment by syntaxlexx** [comment]
https://github.com/acelords/laravel-messenger-demo-inertia

The Laravel Inertia demo is finally here!! React NAtive coming up next!
